main/options.c

```diff
@@ -502,6 +503,9 @@ static const char *const Features [] = {
 	"json",
 	"interactive",
 #endif
+#ifdef HAVE_SECCOMP
+        "secure",
```

Comment:
I don't know about c-guess, but it seems that it failed to guess correctly.
If you use EditorConfig, it will set the tab settings correctly.

We already have settings for Uncrustify and GNU indent. I don't know we should support clang-format besides them (or switch to it).